test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -384,15 +407,18 @@ type createPodsOp struct {
 	// Optional
 	PersistentVolumeTemplatePath      *string
 	PersistentVolumeClaimTemplatePath *string
+	// When true, the given number of pods get created, scheduled and
+	// deleted one after the other. This can be used to determine
+	// the latency for getting a single pod running instead of
+	// pod scheduling throughput.
+	Sequentially bool
```

Comment:
Can you use the latency metrics instead? In particular `pod_scheduling_duration_seconds`

Reply:
Yes, `pod_scheduling_duration_seconds` seems better than observing the same thing in the client (`PodLatency`). That will make the code quite a bit simpler. Just to be sure before I do that, `pod_scheduling_duration_seconds` tracks the entire time for a pod also across multiple scheduling attempts, right?

The numbers match, so it looks like it does.

Reply:
I removed `PodLatency`. But `createPodsSequentially` remains because creating pods in bulk vs. one-after-the-other are different kind of workloads.

Reply:
But doing it via qps then is just rate limiting. It doesn't create the next pod as soon as the old one is scheduled.

That answers the question "can I schedule x pods/s
?". It does not answer the question "on average, how long does it take to schedule one pod?". For that, clusterloader would have to increase the rate until the cluster doesn't keep up anymore.

Reply:
> on average, how long does it take to schedule one pod?

You shouldn't need to create pods sequentially to measure this. That's why we have latency metrics. Creating pods sequentially sounds very artificial (detached from reality in production).

A more appropriate question is how long do pods take to schedule on average at a given pod creation rate.

Reply:
We seem to be interested in different answers. When I was working on reducing latency, it was easier to deal with scheduling of one pod at a time. To achieve that with a fixed pod creation rate, that rate would have to be low enough to not accidentally create the next pod too soon, so most of the time it'll run more slowly than a test that creates the next pod as soon as possible.

I can keep this in a private branch if no-one else is interested in this.

Reply:
I just think it's too artificial to be worth

Reply:
users in a real cluster don't wait for other pods to be scheduled before creating their own.